docs/content/policy-reference.md

```diff
@@ -337,6 +337,7 @@ The following table shows examples of how ``glob.match`` works:
 | -------- | ---------- | ----------- |
 | ``output := glob.match("*.github.com", [], "api.github.com")`` | ``true`` | A glob with the default ``["."]`` delimiter. |
 | ``output := glob.match("*.github.com", [], "api.cdn.github.com")`` | ``false`` | A glob with the default ``["."]`` delimiter. |
+| ``output := glob.match("*.github.com", null, "api.cdn.github.com")`` | ``true`` | A glob without delimiter. |
```

Comment:
Can we make the example a little more obvious, please?  Maybe match with pattern `*hub.com`?